src/awst_build/eb/op-module-builder.ts

```diff
@@ -33,7 +35,7 @@ export class IntrinsicOpGroupBuilder extends NodeBuilder {
     }
     const metaData = this.opGrouping.ops[name]
 
-    if (metaData.signatures.some((s) => s.argNames.length)) {
+    if (VOID_OPS.includes(metaData.op) || metaData.signatures.some((s) => s.argNames.length)) {
```

Comment:
Could we not just check the return type?
```suggestion
    if (VOID_OPS.includes(metaData.op) || metaData.signatures.some((s) => s.argNames.length || s.returnType.equals(voidPType))) {
```

Reply:
Good point. It has been updated. 3c6132d